<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>person</class>
 <widget class="QWidget" name="person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Image/background.jpg);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>申请</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <widget class="QPushButton" name="pb_apply_budaka">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>10</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
           </property>
           <property name="text">
            <string>申请补打卡</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_apply_vacation">
           <property name="geometry">
            <rect>
             <x>590</x>
             <y>10</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
           </property>
           <property name="text">
            <string>申请休假</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_apply_jiaban">
           <property name="geometry">
            <rect>
             <x>700</x>
             <y>10</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
           </property>
           <property name="text">
            <string>申请加班</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_delete">
           <property name="geometry">
            <rect>
             <x>800</x>
             <y>10</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
           </property>
           <property name="text">
            <string>删除选中项</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="tableWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>871</width>
             <height>192</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
           </property>
           <column>
            <property name="text">
             <string>姓名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>日期</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>类型</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>员工说明</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>状态</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>个人设置</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>个人设置</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>当前密码</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="le_name">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="currentpsw">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>170</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>210</y>
          <width>54</width>
          <height>12</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>新密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="newpsw">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>210</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>250</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>确认新密码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="querenpsw">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>250</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_save">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>320</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_return">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>320</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0.5);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
       <widget class="QLabel" name="lb_currentpsw">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>170</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lb_queren">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>250</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="lb_show">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>370</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #4288CE;
background: rgba(154, 207, 206,0);
color: rgb(255, 255, 255);
border:none;
border-radius:7px;
cursor:pointer;
border-image:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>退出</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
